PinpointKit/PinpointKit/Sources/SystemLogCollector.swift

```diff
@@ -6,12 +6,30 @@
 //  Copyright © 2016 Lickability. All rights reserved.
 //
 
+/**
+ The type of logs to collect.
+ 
+ - Application: Logs from the application target.
+ - Testing:     Logs from the testing target.
+ */
+public enum ASLLoggingType {
```

Comment:
Also, maybe this should be a subtype of `SystemLogCollector`. It's only used in that context, and we may be able to drop `ASL` entirely, since clients probably don't have to worry that the logging type is the ASLLogger type under the hood.